components/template/Templates.vue

```diff
@@ -27,10 +27,13 @@
           class="my-2"
           :class="{ 'd-flex align-items-center': !showFullDescription }"
         >
-          <p class="content-description mt-0 mb-1">
-            {{ activeCategory.description }}
-          </p>
+          <p
+            ref="description"
+            class="content-description mt-0 mb-0"
+            v-html="activeCategory.description"
```

Comment:
Reason for using v-html?

Reply:
Because the description is html, as we have implemented the editor on the modal where we are adding the template category

Reply:
Ok then you can self close the p tag